apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.php

```diff
@@ -34,6 +34,7 @@ function get_menu_params_by_theme_location( $location ) {
 			$params = [
 				'theme_location' => 'menu-1',
 				'menu_class'     => 'main-menu',
+				'container'       => 'ul',
```

Comment:
What's the purpose of this (at least, I can't see any markup differences)?..

Reply:
https://developer.wordpress.org/reference/functions/wp_nav_menu/#parameters The default is `div` but maybe TwentyNineteen is already overriding this?

Reply:
adding this was the only way I could get main-menu added to the nav menu `ul` which is what the existing css was expecting. It seems like `'items_wrap'     => '<ul id="%1$s" class="%2$s">%3$s</ul>',` should do that, but this param doesn't seem to have any impact.

Reply:
Just noticed that this fix is actually making the element nesting different, so will take another look at this.

Reply:
There's an extra `div` container around the `ul` in vanilla TwentyNineteen

<img width="1072" alt="vanilla-published" src="https://user-images.githubusercontent.com/1270189/60550297-5daf0480-9cdc-11e9-8b1e-f3eeb73f5099.png">
Vanilla, published markup (I named my menu foobar)

<img width="1048" alt="plugin-published" src="https://user-images.githubusercontent.com/1270189/60550322-76b7b580-9cdc-11e9-800e-cfdd58177241.png">
FSE published

Reply:
We can leave this for a followup, but it looks like we're missing a header element in the FSE implementation.

Reply:
> We can leave this for a followup, but it looks like we're missing a header element in the FSE implementation. Added #34423

Probably we should add the missing header in our page FSE template ([#](https://github.com/Automattic/wp-calypso/blob/85fc28d75538f1a973f04158187272b32e523464/apps/full-site-editing/full-site-editing-plugin/full-site-editing/page-fse.php#L23)) or when getting the header content from the template part ([#](https://github.com/Automattic/wp-calypso/blob/85fc28d75538f1a973f04158187272b32e523464/apps/full-site-editing/full-site-editing-plugin/full-site-editing/utils/class-a8c-wp-template-data-inserter.php#L73-L78)).

Reply:
The issue here was that I didn't have the menus added to my site, so the menu was falling back to a page menu, which has a different dom structure - have added another issue for this, and in this PR have stuck to fixing styling when site menus are in place. Was able to remove the ul container param in this case and things displayed as expected.

Reply:
Have added a separate issue for dealing with instances were site menus are not defined #34429

Reply:
Menus, amirite?!